exercises/concept/lasagna/src/main/java/Lasagna.java

```diff
@@ -1,9 +1,18 @@
+
 public class Lasagna {
-    // TODO: define the 'expectedMinutesInOven()' method
+    public static int expectedMinutesInOven() {
+        return 40;
+    }
 
-    // TODO: define the 'remainingMinutesInOven()' method
+    public static int remainingMinutesInOven(int timeInOven) {
+        return expectedMinutesInOven() - timeInOven;
+    }
 
-    // TODO: define the 'preparationTimeInMinutes()' method
+    public static int preparationTimeInMinutes(int layers) {
+        return layers * 2;
+    }
 
-    // TODO: define the 'totalTimeInMinutes()' method
+    public static int totalTimeInMinutes(int amountLayers) {
+        return preparationTimeInMinutes(amountLayers) + expectedMinutesInOven();
+    }
```

Comment:
The goal of this exercise is to have students learn how to define methods. Putting the actual implementation in them would defeat that purpose.